types/cytoscape-dagre/tsconfig.json

```diff
@@ -0,0 +1,24 @@
+{
+    "compilerOptions": {
+        "module": "commonjs",
+        "lib": [
+            "es6",
+            "dom"
```

Comment:
just question, why you've added `dom` to the `lib`? (not obvious from the code)
thx!

Reply:
thx!